packages/web_framework/tests/web

```diff
@@ -0,0 +1 @@
+../../../test/unit/web
```

Comment:
I think we should have `packages/web_framework/tests/web_framework` pointing to `../../../test/unit/web_framework` instead.